config/routes.rb

```diff
@@ -48,7 +48,8 @@
   as :user do
     # :sessions
     get 'sign_in' => 'devise/sessions#new', as: :new_user_session
```

Comment:
this now needs to be `user/sessions#new` -- otherwise the memorised uid in the session is never removed